### ISSUE TITLE
Is it possible to add regular text hyperlinks to the text, which will be preserved when exporting to PDF?

### ISSUE THREAD

**Issue by zhartus** [uxbox/uxbox#1381]
**Is your feature request related to a problem? Please describe.**
It is often necessary to add regular hyperlinks inside a text block. Now I use a transparent block for this and on-click prototype intereaction. But this is kind of a dirty hack, and besides, it only works in view mode. When exporting to pdf, even such links are lost. 

**Describe the solution you'd like**
Just classic text hyperlinks like everywhere else.

**Comment by hirunatan** [comment]
Hello. This should be an useful enhancement. It's not clear how would be implement it, but it can be considered.

We alredy had a planned enhancement related to this. I've added your suggestion there:
https://tree.taiga.io/project/penpot/us/2327

**Comment by zhartus** [comment]
@hirunatan Great, thanks.

Just in case, I just want to clarify that in this case I mean just simple text links inside text blocks. The enhancement you specified seems to be talking about the existing links in the prototyping tool, which are proposed to be transferred to pdf. But there are no ordinary text links yet, even in the prototyping tool, there for now you can assign a link only to the entire block as a whole, but not to the selected part of the text.

**Comment by hirunatan** [comment]
Yes. I understand that there are two different topics, but they probably may be studied together. This the reason for me adding it to this issue. Thanks for your suggestions.

**Comment by the94air** [comment]
Hi. I am looking for any workarounds that allow adding hyperlinks but without having to convert the exported pdf to word. Is it simply not possible?

**Comment by Thatoo** [comment]
Hello, any update on this issue?

**Comment by Thatoo** [comment]
Hello, may I ask if this feature is in any roadmap?
